FinderCompass/src/main/java/atomicstryker/findercompass/common/DefaultConfigFilePrinter.java

```diff
@@ -204,4 +217,56 @@ else if (settingList.isEmpty())
             }
         }
     }
+    
+	private Map<String,CompassTargetData> getBlocks(String pBlockName, int pDamage) {
+		Map<String, CompassTargetData> data = new HashMap<String, CompassTargetData>();
+
+		Block block = GameData.getBlockRegistry().getObject(pBlockName);
+		if (block != Blocks.air) {
+			String oreDictName = null;
+			CompassTargetData compassTargetData =null;
+			
+			int[] blockIDs = OreDictionary.getOreIDs(new ItemStack(block, 1, pDamage));
+			for (int oreBlockID : blockIDs) {
+				oreDictName = OreDictionary.getOreName(oreBlockID);
+				compassTargetData = data.get(oreDictName);
+				if (compassTargetData == null){
+					compassTargetData = new CompassTargetData(oreDictName);
+				}
+				
+				for (ItemStack stack : OreDictionary.getOres(oreDictName)) {
+					Item item = stack.getItem();
+					int damage = item.getDamage(stack);
+					block = GameData.getBlockRegistry().getObject(item.delegate.name());
+					compassTargetData.add(block, damage);
+				}
+				
+				if (!compassTargetData.isEmpty()){
+					data.put(oreDictName, compassTargetData);
+				}
+			}
+			
+			if (compassTargetData == null){
+				compassTargetData = new CompassTargetData(block, pDamage, oreDictName);
+				if (!compassTargetData.isEmpty()){
+					data.put(oreDictName, compassTargetData);
+				}
+			}
+		} else {
+			String oreDictName = pBlockName;
+			CompassTargetData compassTargetData = new CompassTargetData(oreDictName);			
+			for (ItemStack stack : OreDictionary.getOres(oreDictName)) {
+				Item item = stack.getItem();
+				int damage = item.getDamage(stack);

```

Comment:
You put all meta values in? I don't know enough about the Oredict, but the function does take meta arguments - can there be different (meta) blocks behind an oredict name?